### DISCUSSION TITLE
1º Encontro FrontendBR (BrazilJS)

### DISCUSSION THREAD

**Discussion by eduardojmatos** [frontendbr/forum#287] [Discussão]
**Olá pessoal!**

Como várias pessoas daqui vão participar da BrazilJS, [conforme discutimos aqui](https://github.com/frontendbr/forum/issues/278), decidimos organizar nosso primeiro Encontro FrontendBR!

Vai ser uma ótima oportunidade de nos conhecermos pessoalmente!

O que acham? Quem vamos? Provavelmente será na sexta-feira, após a conferência!

*comente se você for pra podermos decidir um local que caiba todo mundo!

**Comment by notnotgabriel** [comment]
Simmm! Bora lá :dancer:

**Comment by LulzAugusto** [comment]
🏄 LGTM

**Comment by femontanha** [comment]
o/

**Comment by felipefialho** [comment]
Queria :cry:

**Comment by zehfernandes** [comment]
Estarei lá :)

**Comment by titxoo** [comment]
Tamo junto :metal:

**Comment by fdaciuk** [comment]
Vou :dancer: :dancer: :sparkles: :shipit:

**Comment by vagnervjs** [comment]
bora :metal:

**Comment by danguilherme** [comment]
Buera! :rocket:

**Comment by vitornp** [comment]
Boraaa

Se liga ai @agcorrea

**Comment by eduardojmatos** [comment]
10 pessoas já! :grinning:

Convoco o @icaromh que manja dos paranuês dos rolês de POA!

**Comment by agcorrea** [comment]
Bora!

**Comment by talitapagani** [comment]
Conte comigo!

**Comment by victormiguez** [comment]
Partiu!

**Comment by icaromh** [comment]
@eduardojmatos eu to PUTO DA CARA. Vou em um casamento em SC no finde do BrazilJS! 

Falei pra minha prima mudar a data e ela não quis...

**Comment by luchesigui** [comment]
🚶 simbora 🚶

**Comment by renatogalvones** [comment]
:metal:

TAMO JUNTO!

Vou eu e mais 3 caboclos que não estão aqui no tópico!

**Comment by felipebernardes** [comment]
demorouuuuuuu @eduardojmatos ! já temos um bar? :beer: :beers: 

Pra quem vai de uber pro encontro, vou postar aqui o ponto de encontro lá no barra shopping. Assim, dá pra dividir a corrida e economizar.

o/

**Comment by yanmagale** [comment]
Siiiimbooora

**Comment by hugodutra-zz** [comment]
Opa! Partiu!

cc: @alanhr

**Comment by ramonsenadev** [comment]
Go Go Go

**Comment by victorferraz** [comment]
animo

**Comment by danguilherme** [comment]
@eduardojmatos, cria o evento no https://github.com/frontendbr/eventos? :smile:

Aliás, não tem BrazilJS lá?! :open_mouth:

**Comment by mvfsillva** [comment]
Vamos nessa!

**Comment by obetomuniz** [comment]
Opa, estou dentro! 🎸

**Comment by reinaldoferraz** [comment]
Sim!

**Comment by eduardojmatos** [comment]
@danguilherme boa ideia, só vamos esperar mais pessoas se interessarem pra definirmos o local!

**Comment by willianjusten** [comment]
Caramba! Quanta gente já ahuahuahuahua

Eu to junto! Aproveitar que vai ser também minha despedida <3

**Comment by willycamargo** [comment]
Vão fechar uma churrascaria?

**Comment by felipefialho** [comment]
Vai rolar despedida de solteiro pro @willianjusten ?

**Comment by willianjusten** [comment]
@LFeh, para essa aí acontecer ainda vai demorar muito ahauhuauha
Vai ser só despedida do Brasil mesmo xD

**Comment by fernandomoraes** [comment]
TO JUNTO!

**Comment by rdiego26** [comment]
Bora!
:metal:

**Comment by eduardojmatos** [comment]
@willycamargo pode ser sim, ou um boteco com alguns petiscos tbm rolaria! =D

**Comment by willycamargo** [comment]
Arrumem o local. Só não vou se ficar muito fora de mão mesmo. Mas se não for muito longe do  evento ou do hotel. estou dentro!

**Comment by fernandomoraes** [comment]
Alguem vai ficar perto do Theatro Sao Pedro?

**Comment by willycamargo** [comment]
Vou ficar no Ibis, fica a 13 quadras do Theatro São Pedro, @fernandomoraes

**Comment by danguilherme** [comment]
Eita! Vou ficar a uma quadra desse teatro! rs

![image](https://cloud.githubusercontent.com/assets/1488378/17906359/56ef688a-694d-11e6-8dba-c3bab26aed7e.png)

**Comment by fernandomoraes** [comment]
Marcar umas cervas!

**Comment by ramonsenadev** [comment]
Mais alguém hospedado no Coral Tower Express?

Em 23 de ago de 2016 4:19 PM, "Guilherme Ventura" notifications@github.com
escreveu:

**Comment by cjpatoilo** [comment]
Opa! To na área, próximo ao Hospital Dom Vicente Scherer.

**Comment by eduardojmatos** [comment]
Vou ficar no Ibis tbm

**Comment by fernandomoraes** [comment]
show de bola, algum lugar ai na região pra marcar?

**Comment by ghost** [comment]
Opa, estarei no Coral Tower Express.

Estarei chegando na quinta @ramonsenadev ;)

E @eduardojmatos só marcar lugar, que vou hein! Ano passado fomos para um lugar chamado "TocaDaCoruja" mas lá é um pouco pequeno. Mas é bem dahora!

**Comment by willycamargo** [comment]
Eu chego na quinta-feira após o almoço (umas 15h ~ 16h). Algo marcado para quinta?

**Comment by eduardojmatos** [comment]
Acham justo falarmos via Telegram ou Whatsapp? Ou Slack

**Comment by eduardojmatos** [comment]
Ops, tinha fechado sem querer :sob:

**Comment by willycamargo** [comment]
Justo. Mande o link do slack aqui para o pessoal =)

(meu email agradece hahaha)

**Comment by ghost** [comment]
Justíssimo, pode ser criar uma sala lá no braziljs.slack.com

manda ai pra a gente o link.

**Comment by eduardojmatos** [comment]
No slack da BrazilJS ou FrontendBrasil?

**Comment by ghost** [comment]
Acho que é melhor FrontEndBrasil mesmo, faz mais sentido :)

**Comment by eduardojmatos** [comment]
Pra quem não tem: http://frontendbrasil-slack.herokuapp.com/

**Comment by fernandomoraes** [comment]
grupo de whats fica mais pratico, acho

**Comment by eduardojmatos** [comment]
@fernandomoraes tem toda razão! O Slack não deixa criar channel privado com mais de 8 pessoas :

Bom, me adicionem então: 19 98142-9773

**Comment by fernandomoraes** [comment]
@eduardojmatos foee

**Comment by eduardojmatos** [comment]
@icaromh mas ajuda a gente então :( não manjo dos rolês bacanas pra gente fazer aí

**Comment by willycamargo** [comment]
43 99544614

**Comment by eduardojmatos** [comment]
@fernandomoraes @willycamargo @vagnervjs já add lá!

**Comment by agcorrea** [comment]
19 988043936

**Comment by dqueiroz** [comment]
saindo amanhã do interior do MT para pegar o vôo quinta feira em Goiânia para POA - 66996098686

**Comment by ramonsenadev** [comment]
71 996860484

**Comment by Lucasktrindade** [comment]
Opa, show, acho justo um encontro, pode me adicionar no grupo, meu número é 21 980135232

**Comment by diogomoretti** [comment]
Opa! Imagina se vou perder. Chego na quinta a noite já... vou ficar pela cidade baixa. Quem tá criando o grupo do Telegram/WPP?

@eduardojmatos

**Comment by eduardojmatos** [comment]
Opa, já criei, me manda seu tel!

**Comment by vameri** [comment]
51 93573343 +1  @vameri

**Comment by ghost** [comment]
85 987215077

**Comment by cyz** [comment]
Partiu! Meu telegram: @icynthia

**Comment by eduardojmatos** [comment]
@icynthia pessoal decidiu usar Whatsapp, me passa seu número pra eu add aqui! :)

**Comment by cyz** [comment]
@eduardojmatos eu não uso Whatsapp. e isso é sério hahaha
bem, qualquer tiver um local será divulgado em fb ou em algum lugar?

**Comment by Lucasktrindade** [comment]
@eduardojmatos  21 980135232

**Comment by jaydson** [comment]
Que massa isso!
Quero participar hein

**Comment by eduardojmatos** [comment]
@cyz blz! Vamos criar um evento no fb logo mais :)

**Comment by eduardojmatos** [comment]
@jaydson boa!! Vamos fechar o lugar até amanhã e divulgar por aqui! :)

**Comment by willianjusten** [comment]
Começo a achar que precisaremos ir para algum lugar bem grande ahuahuahu

**Comment by mandado** [comment]
@eduardojmatos 32 984995069

**Comment by alexfigueiredo** [comment]
14 98810 1930

**Comment by felipebernardes** [comment]
@willianjusten isso seria um ótimo problema!

**Comment by hugodutra-zz** [comment]
me adicionem ae 21 98746-8402. Meu telegram é hugoadrj

**Comment by luchesigui** [comment]
19 98389-3232

vale pra whats e telegram haha

**Comment by eduardotecnologo** [comment]
tou dentro...Bora 👍 ;)
zap (32)98702-4149

**Comment by notnotgabriel** [comment]
@eduardojmatos me add lá
19 99350 8940

**Comment by eduardojmatos** [comment]
Galere, criamos o evento privado, só pra gente controlar a quantidade de pessoas.

Vocês podem me mandar mensagem no Facebook pra eu adicioná-los?

https://www.facebook.com/eduardoj.matos

**Comment by thulioph** [comment]
Tô dentro e @renatooliveira também 😄

**Comment by IanRamosC** [comment]
Tou dentro. @alisonmonteiro e @matheuss também.

**Comment by eduardojmatos** [comment]
@thulioph e @IanRamosC me chamem no Facebook pra eu adicionar todo mundo certinho =D

**Comment by tcelestino** [comment]
Injusto para quem não vai!!! hahahahaha

No segundo, tem que ser mais para o sudeste, tipo SP! hahaha 

Sucesso e bom encontro/BrazilJS

**Comment by eduardojmatos** [comment]
@tcelestino mancada vc não ir hein!! Mas vamos marcar o próximo quem sabe em SP ou algum lugar do sudeste (RJ, MG, etc.)

**Comment by daniellucindo** [comment]
o/

**Comment by ubirajarapelli** [comment]
Opa! To indo tbm!

**Comment by lagden** [comment]
Tem um barzinho daora em POA http://www.grutaazulclub.com.br
😂

Gostaria muito de ir!! Mas tenho filho e mulher doente!!

Mas aproveitem! 🙌

**Comment by sergiokopplin** [comment]
To dentro junto com o @renatogalvones

**Comment by felipesousa** [comment]
Opa! Bora !

**Comment by fernandoguedes** [comment]
Opa, me convidem pra essa bagaça.

https://www.facebook.com/luisfernandoguedes

48 9675-8485

**Comment by willianjusten** [comment]
Infelizmente não consegui encontrar com vocês, mas fiz um post/video agradecendo pelo carinho de vocês =D

https://willianjusten.com.br/achievement-concluido-braziljs/

**Comment by eduardojmatos** [comment]
Fotenhas que tiramos no encontro! Se tiverem mais, mandem aí!!!

![whatsapp image 2016-08-26 at 23 06 17](https://cloud.githubusercontent.com/assets/1279783/18190480/9865b082-7099-11e6-8e7d-5079a360e446.jpeg)
![whatsapp image 2016-08-26 at 23 07 36](https://cloud.githubusercontent.com/assets/1279783/18190482/986ddf82-7099-11e6-8d4c-9b7f466c86b2.jpeg)
![whatsapp image 2016-08-26 at 23 10 27](https://cloud.githubusercontent.com/assets/1279783/18190481/986ad8aa-7099-11e6-8309-d1a002f25714.jpeg)
